bare/roles/web/files/systemd/mastodon-web.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=mastodon-web
+After=network.target
```

Comment:
Should we have a dependency on the two other services here?

Reply:
I want to stick to the files as they are in the production documentation. If we add dependencies, we should do them in the prod documentation first.

Reply:
Works for me